android/app/build.gradle

```diff
@@ -15,9 +15,14 @@ import com.android.build.OutputFile
  *   // the name of the generated asset file containing your JS bundle
  *   bundleAssetName: "index.android.bundle",
  *
- *   // the entry file for bundle generation
+ *   // the entry file for bundle generation. If none specified and
+ *   // "index.android.js" exists, it will be used. Otherwise "index.js" is
+ *   // default. Can be overridden with ENTRY_FILE environment variable.
  *   entryFile: "index.android.js",
  *
+ *   // https://reactnative.dev/docs/performance#enable-the-ram-format
```

Comment:
The `#enable-the-ram-format` does not exist, does it?

Reply:
Hm not my fault :D